src/components/dark-theme/useDarkMode.tsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react';
+
+const useDarkMode = () => {
+  const [theme, setTheme] = useState('light');
+
+  const setCookie = (name : string, value: string) => {
+    const domain = '.realdevsquad.com';
+    const expires = new Date(Date.now() + 24 * 30 * 60 * 60 * 1000);
+    document.cookie = `${name}=${value}; expires=${expires}; domain=${domain}; path=/`;
```

Comment:
${name} will be always `theme`, so could we just use it here directly and not use it as an argument.

Reply:
Will do the required changes.